### PR TITLE
Leaderboard

### PR DESCRIPTION
**Pull request by fywu85**
Fix minor bugs in previous PR. This one is tested with and passed `docker build`.

### CODE REVIEW

File: flow/utils/leaderboard/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	apt-get -y upgrade && \
+	apt-get install -y \
+    vim \
+    apt-utils
+
+# Flow
+RUN cd ~ && \
+	git clone https://github.com/flow-project/flow.git && \
+    cd flow && \
+    git checkout leaderboard && \
```
Comment (nskh) [inline flow/utils/leaderboard/Dockerfile]: If we're merging this into master, is the `git checkout leaderboard` going to be necessary?

Comment (fywu85) [reply flow/utils/leaderboard/Dockerfile]: My plan is to keep the leaderboard branch as sort of devel branch and only push stable, tested codes to master once in a while. I just realized that I failed all the checks, again... Let me fix this!

Comment (AboudyKreidieh) [reply flow/utils/leaderboard/Dockerfile]: switch to master

File: flow/utils/leaderboard/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	libxml2-dev \
+	libxslt1-dev \
+	openjdk-8-jdk
+
+# SUMO
+RUN cd ~ && \
+	git clone --recursive https://github.com/eclipse/sumo.git && \
+	cd sumo && \
+	#git checkout b6f37dd && \
+	git checkout cbe5b73 && \
```
Comment (AboudyKreidieh) [inline flow/utils/leaderboard/Dockerfile]: can u fix to `016c09d306`?

**Comment by fywu85** [comment]
@AboudyKreidieh This should be good to go.

**Pull request merged**